types/pretty/pretty-tests.ts

```diff
@@ -0,0 +1,5 @@
+import { pretty } from "pretty";
```

Comment:
The underlying library does not appear to export a property called "pretty".  Should this be `import pretty = require("pretty");`?

Reply:
@amcasey Good pickup, I overlooked the testing requirements - fixed.